ui/helpers/higher-order-components/authenticated/authenticated.component.js

```diff
@@ -1,12 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Redirect, Route } from 'react-router-dom';
-import { UNLOCK_ROUTE, INITIALIZE_ROUTE } from '../../constants/routes';
+import {
+  UNLOCK_ROUTE,
+  INITIALIZE_ROUTE,
+  ONBOARDING_ROUTE,
+} from '../../constants/routes';
 
 export default function Authenticated(props) {
   const { isUnlocked, completedOnboarding } = props;
-
   switch (true) {
+    case process.env.ONBOARDING_V2 === true:
```

Comment:
This seems like it should be below, alongside the `completedOnboarding` case :thinking: 
Why would we redirect users here before they've completed onboarding?

Reply:
@Gudahtt honestly this was added in for development convenience purposes. But I can make the change locally while in dev without committing it!

Reply:
Ah OK. I'm not opposed to adding temporary stuff like this since it's behind the flag, but a comment to help identify these cases would be great.

Reply:
Gotcha! Yeah I really ought to have called that out! Will add a comment!